Documentation/cmdref/cilium_service_update.md

```diff
@@ -11,6 +11,7 @@ cilium service update [flags]
 ### Options
 
 ```
+      --backend-weights uints       Backend weights (0 means maintenance) (default [])
```

Comment:
This commit can probably be squashed with the one that introduced the new CLI option